### PR TITLE
Fix update method failure with known record types

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/3596

## Checklist
- [x] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [x] Added tests

### CODE REVIEW

File: ballerina/tests/complextype_client.bal [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import ballerina/sql;
```
Comment (niveathika) [inline ballerina/tests/complextype_client.bal:1]: Missing License

Comment (kaneeldias) [reply ballerina/tests/complextype_client.bal:17]: Fixed in [c449c73](https://github.com/ballerina-platform/module-ballerina-persist/pull/57/commits/c449c733f56de2eb848c6bc8d06176694ffb55ce)

**Review by niveathika — APPROVED** [review]

File: ballerina/utils.bal [hunk 2]
```diff
@@ -38,7 +39,15 @@ isolated function flattenRecord(record {} r) returns record {} {
     return returnRecord;
 }
 
+isolated function isKnownRecordType(record {} r) returns boolean {
+    if r is time:Civil || 
+       r is time:TimeOfDay || 
+       r is time:Date {
```
Comment (daneshk) [inline ballerina/utils.bal:45]: what if we have other types like `uuid:Uuid` order than the time module? any type coming from a different module.

Comment (kaneeldias) [reply ballerina/utils.bal:45]: Seems the only known record types we support in `sql:Value` is the time module types.
https://github.com/ballerina-platform/module-ballerina-sql/blob/master/ballerina/types.bal#L36

**Pull request merged**